tests/integration/update_cluster/aws-lb-controller/kubernetes.tf

```diff
@@ -506,12 +506,18 @@ resource "aws_launch_template" "nodes-minimal-example-com" {
   user_data = filebase64("${path.module}/data/aws_launch_template_nodes.minimal.example.com_user_data")
 }
 
-resource "aws_route" "route-0-0-0-0--0" {
+resource "aws_route" "route-ipv4-default" {
```

Comment:
This change means that existing clusters will have their route deleted and recreated which would result in a network outage. I've tried to make this exact change before and terraform ended up failing because it tried to create the new route before deleting the old route, causing the `apply` to fail and having to rerun `apply` a second time to create the replacement route. During that period of time the route was missing.

Alternatively we document yet another `terraform state mv` that is needed when upgrading kops

Reply:
Ah... well.... I guess the IPv6 one will be named ----0 for Terraform.